tests/testthat/test-utils.R

```diff
@@ -13,3 +13,10 @@ test_that("is_file_local works",{
   expect_false(is_file_local(test_path("data")))
 
 })
+
+test_that("to_list works as expected", {
```

Comment:
Instead of that, add additional tests in `test-ddi.R` that pass perpetrator as vector or list.